end_users/README.md

```diff
@@ -1,3 +1,24 @@
 # End Users
 
-Documentation coming soon...
+## Table of Contents
+
+* [Merge Options](#merge-options)
+  * [Setting the Correct Merge Option](#setting-the-correct-merge-option)
+
+## Merge Options
+
+Repositories that use BonnyCI should not use any merge strategy which changes the SHA of the commit. This is because [Zuul](https://github.com/openstack-infra/zuul), the scheduler used in BonnyCI, will be thrown off by the change in SHA when it merges commits that successfully pass the gate queue.
```

Comment:
@mlangbehn: maybe. I'm kind of against holding this up to wait on us writing documentation outlining most of what's included [here](https://github.com/BonnyCI/hoist/tree/master/roles). I'm also not sure if that documentation belongs in the end-user section.

We could always deliver that at in the near future and update this documentation accordingly.